tests/pytest_framework/src/syslog_ng_config/syslog_ng_config.py

```diff
@@ -81,15 +81,31 @@ def create_filter_group(self, filters):
         self.__syslog_ng_config["filters"].update(config_group.full_group_node)
         return config_group
 
-    def create_logpath(self, sources=None, filters=None, destinations=None, flags=None):
+    def create_logpath(self, sources=None, filters=None, destinations=None, logpaths=None, flags=None):
         logpath = LogPath()
         if sources:
             logpath.add_source_groups(sources)
         if filters:
             logpath.add_filter_groups(filters)
         if destinations:
             logpath.add_destination_groups(destinations)
+        if logpaths:
+            logpath.add_logpaths(logpaths)
         if flags:
             logpath.add_flags(flags)
         self.__syslog_ng_config["logpaths"].update(logpath.full_logpath_node)
         return logpath
+
+    def create_inner_logpath(self, sources=None, filters=None, destinations=None, logpaths=None, flags=None):
```

Comment:
Can you separate this patch too in a separate PR? I miss an example here as well.

Reply:
+1, separate PR is coming